library/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateVnetStep.java

```diff
@@ -73,20 +72,32 @@ private Network createVnetAndSubnets(
           .withAddressSpace(
               parametersResolver.getValue(
                   CromwellBaseResourcesFactory.ParametersNames.VNET_ADDRESS_SPACE.name()))
-          .withSubnet(
-              CromwellBaseResourcesFactory.Subnet.AKS_SUBNET.name(),
+          .defineSubnet(CromwellBaseResourcesFactory.Subnet.AKS_SUBNET.name())
+          .withAddressPrefix(
```

Comment:
Why do we need this?

Reply:
We need to use definteSubnet instead of withSubnet to attach NSGs.